product-cli/cmd/cluster/cluster.go

```diff
@@ -0,0 +1,44 @@
+package cluster
+
+import (
+	"github.com/openshift/hypershift/api/v1beta1"
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/hypershift/cmd/cluster/agent"
+	"github.com/openshift/hypershift/cmd/cluster/core"
+	"github.com/openshift/hypershift/cmd/log"
+)
+
+func NewCreateCommands() *cobra.Command {
+	opts := &core.CreateOptions{
+		Namespace:                      "clusters",
+		Name:                           "example",
+		ReleaseImage:                   "",
+		PullSecretFile:                 "",
+		ControlPlaneAvailabilityPolicy: "SingleReplica",
```

Comment:
Let's default to HA here for the product CLI :)

Reply:
Ok, it's been updated.